tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -78,7 +79,8 @@ private IPublicClientApplication CreatePca()
                 .Create(clientId)
                 .WithAuthority(this.authorityCbx.Text)
                 .WithExperimentalFeatures(true)
-                .WithBroker(this.useBrokerChk.Checked)
+                //.WithBroker(this.useBrokerChk.Checked)
+                .WithWindowsBroker(this.useBrokerChk.Checked)
                 // there is no need to construct the PCA with this redirect URI, 
                 // but WAM uses it. We could enforce it.
                 .WithRedirectUri($"ms-appx-web://microsoft.aad.brokerplugin/{clientId}")
```

Comment:
> .WithRedirectUri($"ms-appx-web://microsoft.aad.brokerplugin/{clientId}") [](start = 15, length = 73)

are we doing this by default with WAM and uwp?, I think we should

Reply:
No, we decided to keep the old model where the redirect uri is used by fallback browser (e.g. in case same code runs on Linux)